packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1026,7 +1026,7 @@ describe('shallow', () => {
       expect(wrapper.first('div').text()).to.equal('yolo');
     });
 
-    it('should call componentWillReceiveProps, shouldComponentUpdate and componentWillUpdate with merged newProps', () => {
+    it('should call componentWillReceiveProps, shouldComponentUpdate and componentWillUpdate and componentDidUpdate with merged newProps', () => {
```

Comment:
You have a duplicate "and" here (before both the ultimate and penultimate items); also please use an Oxford comma :-)